conf/config-default.yaml

```diff
@@ -104,6 +104,8 @@ apisix:
   #  udp:                         # UDP proxy port list
   #    - 9200
   #    - "127.0.0.1:9201"
+  #  lua_shared_dict:
+  #    lrucache_lock_stream: 10m
```

Comment:
Better to create a new section:
```
stream:
	lua_shared_dict:
		lrucache_lock: 10m
```

So it can be enabled by default. Also, we can add the stream suffix by default.

Reply:
okay, I'll try